Sources/LocalReceiptParsing/Builders/ASN1ContainerBuilder.swift

```diff
@@ -151,7 +151,10 @@ private extension ASN1ContainerBuilder {
 
         if lengthDefinition == .indefinite {
             innerContainers = try buildInternalContainers(payload: data.dropFirst(bytesUsedForLength))
-            let innerContainersOverallLength = innerContainers.map { $0.totalBytesUsed }.reduce(0, +)
+            let innerContainersOverallLength = innerContainers
+                .lazy
```

Comment:
This avoids creating an array of N elements just to calculate the sum.

Reply:
I didn't know about this lazy thing. For whoever is interested this article explains it very well https://www.avanderlee.com/swift/lazy-collections-arrays/

Reply:
Since this is not very well known, maybe adding a comment with a explanation would be worth it? Something like, `reduce` is applied without having to create an intermediate collection with `map`

Reply:
Done